components/omega/doc/design/ShallowWaterOmega0.md

```diff
@@ -1,4 +1,4 @@
-(omega-design-my-class)=
+(omega-design-shallow-water-omega0)=
 # ShallowWaterOmega0
```

Comment:
BTW, I'm not sure how useful this class name is.  Are we going to make such a class or is this just a placeholder?

Reply:
If this isn't a real class name, maybe:
```suggestion
# Omega v0: Shallow Water
```
or something like that?

Reply:
I copied that from the template, but I admit I don't know what it does, or what a class name in markdown is. But just following the pattern, your suggestion is fine.

Reply:
This refers to a C++ class name being documented, not a feature of markdown or the docs themselves. I think we should clarify this in the template. Do you want to add that to this branch, @mark-petersen?